migrations/20200322171627_tags.js

```diff
@@ -0,0 +1,53 @@
+exports.up = async knex => {
+  const buildTableSchema = [
+    {
+      tableName: "tag",
+      create: t => {
+        t.increments("id").primary();
+        t.text("group");
+        t.text("name").notNullable();
+        t.text("description").notNullable();
+        t.boolean("is_deleted").defaultTo(false);
+        t.timestamp("created_at")
+          .notNullable()
+          .defaultTo(knex.fn.now());
+        t.timestamp("updated_at")
+          .notNullable()
+          .defaultTo(knex.fn.now());
+        t.integer("organization_id").notNullable();
+        t.foreign("organization_id").references("organization.id");
+      }
+    },
+    {
+      tableName: "tag_campaign_contact",
+      create: t => {
+        t.increments("id").primary();
+        t.text("value");
+        t.integer("tag_id").notNullable();
+        t.foreign("tag_id").references("tag.id");
+        t.integer("campaign_contact_id").notNullable();
```

Comment:
Please create an index on campaign_contact_id.

Reply:
Just added it but not sure how to test index. Any suggestion?

Reply:
you shouldn't have to test the index.  generally we want to test application features/code -- testing the schema itself isn't as important.